code/app.js

```diff
@@ -0,0 +1,14 @@
+var acc = document.getElementsByClassName("accordion");
```

Comment:
We could improve name of this variable - the most important thing is descriptiveness. Instead of naming it `acc`, we could try something like `accordion`

Reply:
Great point, I will do so from now on.